src/test/java/io/burt/athena/AthenaStatementTest.java

```diff
@@ -494,22 +497,31 @@ void returnsTheValueSetWithSetQueryTimeout() {
         @BeforeEach
         void setUp() {
             queryExecutionHelper.queueStartQueryResponse("Q1234");
+            queryExecutionHelper.queueGetQueryExecutionResponse(QueryExecutionState.QUEUED);
             queryExecutionHelper.queueGetQueryExecutionResponse(QueryExecutionState.SUCCEEDED);
         }
 
-        @Test
-        void setsTheTimeoutUsedForApiCalls1() {
+        @Test()
```

Comment:
Any reason for the added `()` here, or just an editor being helpful? There's another test below with them too.

Reply:
I think I copy-pasted old-style JUnit examples for disabling tests (which added things inside the parentheses, and then I forgot to change it back)